app/serializers/rest/block_serializer.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class REST::BlockSerializer < ActiveModel::Serializer
+  attributes :stealth
+
+  belongs_to :target_account, serializer: REST::AccountSerializer
```

Comment:
So this sets a precedent for rich relationship objects, e.g. if we add equivalent endpoints for follows and mutes, we'll need to keep the objects more or less consistent. Do we want to use `target_account` here? Would `account` not be clear enough?

Reply:
In that context, `account` is probably clear enough. However, if we do add things like `Favourite` etc. (for instance, to differentiate between the time a toot has been fav'ed from the time it has been posted), `account` wouldn't mean the same thing anymore, but maybe that's fine.